freqtrade/exchange/exchange.py

```diff
@@ -772,6 +773,39 @@ def get_fee(self, symbol='ETH/BTC', type='', side='', amount=1,
             raise OperationalException(e) from e
 
 
+# Store Exchange objects created
+__exchanges: Dict[str, Exchange] = {}
```

Comment:
The dict key here can in the future be the `Tuple[str, bool]`, for example, adding space for live/dry-run variants of the exchange objects existing in the same time...

This is just an illustration of the flexibility of the approach...

Reply:
why would you want to have this in the same instance ... ? 

i cannot immagine any scenario where this would make sense...